earth_enterprise/rpms/buildSrc/src/main/groovy/GeeDeb.groovy

```diff
@@ -82,4 +96,19 @@ class GeeDeb extends com.netflix.gradle.plugins.deb.Deb {
 
         findRequires(packageInputFiles).each { requires(it) }
     }
-}
\ No newline at end of file
+
+    protected void formatPackageDescription(String descrip){
```

Comment:
Minor: I was thinking of using a field, but this is acceptable.  E.g., you could define:

```Groovy
void setFreeformDescription(String value)
```

so you can use it like this:

```Groovy
task(type: GeeDeb) {
    freeformDescription = 'Some\nMultiline description.'
}
```

Reply:
Minor: I don't think this should be `protected` it's intended for public use.

Reply:
Interesting. I didn't know fields were assigned in that manner. So it would automatically invoke `setFreeformDescription` when it sees `freeformDescription = 'Some\nMultiline description.'`? 
The only counter-point I would make is that making it a property seems like it involves an extra step later (`packageDescription = freeformDescription`), unless I'm missing something. But maybe using fields is the more Groovy-esque way to do things. Would you prefer that I make it a field? I'll defer to your judgement if you have a preference.